SPECS/python-daemon/python-daemon.spec

```diff
@@ -10,6 +10,8 @@ URL:            https://pypi.python.org/pypi/python-daemon/
 Source0:        https://files.pythonhosted.org/packages/source/p/python-daemon/%{name}-%{version}.tar.gz
 BuildArch:      noarch
 
+%define _python_dist_allow_version_zero 1
```

Comment:
For all such lines, could we add a comment explaining why it needs to be there?

Reply:
sure

Reply:
I found the root cause for the version error in this package and fixed it (had to remove a bogus sed command)